src/components/form/form.scss

```diff
@@ -1,33 +1,30 @@
 @import "./../../style-config/colors";
 
-.carbon-form__save--invalid {
-  background-color: $grey-dark-blue-5;
+.carbon-form__additional-actions {
+  padding-left: 10px;
+  display: inline-block;
 }
 
-.carbon-form__summary {
-  margin: 0 10px;
+.carbon-form__buttons--right {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
 }
 
-.carbon-form__save,
-.carbon-form__cancel {
+.carbon-form-cancel {
   border-radius: 4px;
   display: inline-block;
   margin-top: 20px;
-}
-
-.carbon-form__save {
-  padding: 8px;
-}
-
-.carbon-form__cancel {
   padding: 8px 0 8px 8px;
 }
 
-.carbon-form__additional-actions {
-  padding-left: 10px;
+.carbon-form-save {
+  border-radius: 4px;
```

Comment:
Try removing it and seeing if the buttons still have the border-radius, this looks like it might have been a mistake in the original code